repos/fdbt-site/src/layout/Layout.tsx

```diff
@@ -1,7 +1,7 @@
 import Head from 'next/head';
 import React, { PropsWithChildren, ReactElement, useEffect, useState } from 'react';
 import { Portal } from 'react-portal';
-
+import { myFaresEnabled } from '../constants/featureFlag';
```

Comment:
You can't us feature flags in here because this is all client side

Reply:
removed.